docker-compose.yml

```diff
@@ -22,6 +20,7 @@ services:
     command: ["./docker-wait-for-it.sh", "db:5432", "--", "python3", "manage.py", "runserver", "0.0.0.0:8000"]
     environment:
       DJANGO_SETTINGS_MODULE: "academic_integrity_tool_v2.settings.local"
+      NGROK_DOMAIN: "${NGROK_DOMAIN}"
```

Comment:
It might be useful to abstract this slightly and pass in `ALLOWED_HOSTS` as an environment variable that django could read (I've seen this done as a space separated list that is parsed and appended to the list of allowed hosts).

Reply:
Added an issue, #28, to track this as an enhancement.